tutorials/nnvm/from_darknet.py

```diff
@@ -105,24 +106,45 @@
 
 m.run()
 # get outputs
-out_shape = (net.outputs,)
-tvm_out = m.get_output(0).asnumpy().flatten()
+tvm_out = []
+if MODEL_NAME == 'yolov2':
+    layer_out = {}
+    layer_out['type'] = 'Region'
+    # Get the region layer attributes (n, out_c, out_h, out_w, classes, coords, background)
+    layer_attr = m.get_output(1, tvm.nd.empty((7,), dtype='int')).asnumpy()
+    layer_out['biases'] = m.get_output(2).asnumpy()
+    out_shape = (layer_attr[0], layer_attr[1]//layer_attr[0],
+                 layer_attr[2], layer_attr[3])
+    layer_out['output'] = m.get_output(3).asnumpy().reshape(out_shape)
```

Comment:
Instead of reshape, you can do m.get_output(3).asnumpy()[0]

Reply:
This reshape is needed to make output easy for postprocess.

Reply:
[0] and reshape(1000,) will give the same output since the outshape is (1, 1000)

Reply:
Yolo models output detection boxes with each class probabilities (4 dimension) unlike classical image classification problems which output only class probabilities(imagenet 1000 classes). The image is split into sections and each section will give n number of boxes. So total boxes will be sections times n. These boxes are rearranged to make it easy for postprocess.